hazelcast/src/main/java/com/hazelcast/map/impl/operation/PutAllBackupOperation.java

```diff
@@ -56,8 +56,10 @@ protected void runInternal() {
             // has not been serialized/deserialized
             // and is running directly on caller node
             for (int i = 0; i < recordAndDataValuePairs.size(); i += 2) {
-                Record record = (Record) recordAndDataValuePairs.get(i);
-                putBackup(record);
+                Object obj = recordAndDataValuePairs.get(i);
```

Comment:
Can the record be `null` here since we explicitly check for null before adding it in `PutAllOperation#toBackupListByRemovingEvictedRecords`  and `MergeOperation#toBackupListByRemovingEvictedRecords`?

Reply:
good catch, removed `null` check.